### DISCUSSION TITLE
关于订阅简悦资讯的问题

### DISCUSSION THREAD

**Discussion by LoneFireBlossom** [Kenshin/simpread#3077] [讨论组]
请问开发者，这几个订阅相互重叠吗？比如说我订阅了1⃣️，那Newsletter和其它的是不是全都包括在里面了？
我没看懂所以问一下（主要是不想自己一个一个点开看一下所有文章重合度了……几十分钟没了那就）

```
几个重要的 RSS 订阅源与 Newsletter

📪 简悦 Newsletter

强烈建议订阅，推送频率不会很高，但会以重大更新，与其他生产力联动用法汇总为主。

http://newsletter.simpread.pro/

🗄 简悦 RSS

强烈建议简悦用户订阅

1️⃣ https://rss.simpread.pro/feed → 更新频率高，基本上涵盖了简悦的文章/技巧/玩法/入门/动态/版本更新等内容

2️⃣ https://rss.simpread.pro/zhihu → 来自 简悦官方知乎专栏 的文章

3️⃣ https://rss.simpread.pro/notice → 来自 消息中心（内容来自选项页 → 右下角的消息中心）

4️⃣ https://rss.simpread.pro/plugins → 插件中心 各个插件的介绍

📚 简悦 · 订阅中心

https://simpread.pro/subscribe

#简悦 #简悦小提示 #简悦RSS
```

**Comment by Kenshin** [comment]
问的不错，就留在讨论区吧。

1️⃣  确实包含了余下几个，但侧重点不同，它来自 Telegram Channel 的内容，相当于一个实时通知系统，具有实效性，但因为 Telegram 没发承载太多的文本格式，所以只是个消息系统。

我的建议是：

1. 如果没有特别的需求，可以都添加，比如专门建立个叫做 SimpRead 的文件夹

![image](https://user-images.githubusercontent.com/81074/142566314-a8110650-db07-4c21-9ac4-df4d2f5992d2.png)

2. 反之，订阅 1️⃣   和 2️⃣  以及 **Newsletter**

**Reply by LoneFireBlossom** [reply]
开发者大大能不能再解释一下呀，还是有地方没看懂……
我的需求就是订阅简悦所有的消息，并且不要重复（即不重不漏）。有RSS全文输入最好。那这样的话应该订阅哪几个呢

**Reply by Kenshin** [reply]
哈哈，那就这个吧 https://rss.simpread.pro/feed

一定请同时订阅 http://newsletter.simpread.pro

**Reply by LoneFireBlossom** [reply]
好的，谢谢回复
